lib/app/data/quote_list.dart

```diff
@@ -0,0 +1,269 @@
+// ignore_for_file: lines_longer_than_80_chars
```

Comment:
Let's move this file to utils folder

Reply:
Sure :)

I'm currently working on the `Timer` issue, and once done will be making these changes. Thank you!